docs/content/index.html

```diff
@@ -15,7 +15,35 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width">
-
+	<link rel="schema.dcterms" href="http://purl.org/dc/terms/">
```

Comment:
Suggest inserting HTML comments between tiers to aid visual scanning?

<!-- Dublin Core metadata -->
...
<!-- OpenGraph metadata -->

etc.